week3/week_3_day_1_exercises.Rmd

```diff
@@ -0,0 +1,247 @@
+body %>% 
+  ggplot(data = body, mapping  = aes(x = V24, y = V23)) + 
+  geom_point() + 
+  xlab("Height (in kg)") + 
+  ylab("Weight (in cm)")
+
+```
+
+
+
+-------Labs 3.6.3 (ISL)
+```{r}
+library (MASS)
+library (ISLR2)
+
+
+lm.fit <- lm(medv ~ lstat + age , data = Boston)
+summary (lm.fit)
+
+lm.fit <- lm(medv ~ ., data = Boston)
+summary (lm.fit)
+
+library (car)
+vif (lm.fit)
+
+lm.fit1 <- lm(medv ~ . - age , data = Boston)
+summary (lm.fit1)
+
+```
+
+-------Labs 3.6.4 (ISL)
+
+```{r}
+summary (lm(medv ~ lstat * age , data = Boston))
+```
+
+
+-------Labs 3.6.5 (ISL)
+```{r}
+  
+lm.fit2 <- lm(medv ~ lstat + I(lstat^2), data = Boston)
+summary (lm.fit2)
+
+lm.fit <- lm(medv ~ lstat, data = Boston)
+anova (lm.fit , lm.fit2)
+
+par (mfrow = c(2, 2))
+plot (lm.fit2)
+
+lm.fit5 <- lm(medv ~ poly (lstat , 5), data = Boston)
+summary (lm.fit5)
+
+summary (lm(medv ~ log(rm), data = Boston))
+
+```
+
+
+-------Labs 3.6.6 (ISL)
+```{r}
+head(Carseats)
+
+lm.fit <- lm(Sales ~ . + Income:Advertising + Price:Age, data = Carseats)
+summary (lm.fit)
+
+attach (Carseats)
+contrasts (ShelveLoc)
+
+```
+
+
+
+------6.1 (ISRS)
+```{r}
+babyWeight <- read.delim("babyweights.txt", sep= " ")
+
+
+#part A
+regression_equation <- 123.05 - 8.94 * babyWeight$smoke   
+
+
+#part B
+Mothers_who_smoke <-123.05 - 8.94 * 1
+Mothers_who_dont_smoke <- 123.05 - 8.94 * 0
+
+Diff_In_Weight <- Mothers_who_dont_smoke - Mothers_who_smoke
+Diff_In_Weight
+
+```
+Part C
+
+Reject Null hypothesis  because babies' weight based on mothers who smoke or not are different. The data suggests there's a negative correlation between baby weight and mothers who smoke. 
+
+Reproduce results
+```{r}
+
+babyWeight <- read.delim("babyweights.txt", sep= " ")
+summary(lm(bwt ~ +smoke, data = babyWeight))
+
+```
+
+
+
+
+
+
+-----------6.2 (ISRS)
+```{r}
+babyWeight <- read.delim("babyweights.txt", sep= " ")
+
+#Part A
+regression_equation2 <- 120.07 - 1.93 * babyWeight$parity
+
+#Part B
+first_born <- 120.07 - 1.93 * 0
+not_first_born <- 120.07 - 1.93 * 1
+Diff_In_Weight2 <- first_born - not_first_born
+
+```
+
+Part C
+Fail to reject null hypothesis because there isn't enough data to suggest a weight difference between first born child and other children. 
+
+Reproduce results
+```{r}
+
+babyWeight <- read.delim("babyweights.txt", sep= " ")
+summary(lm(bwt ~ +parity, data = babyWeight))
+
+```
+
+----------6.3 (ISRS)
+
+```{r}
+babyWeight <- read.delim("babyweights.txt", sep= " ")
+#Part A
+babyWeight %>% regression_equation3 <- -80.41 + 0.44(gestation) - 0.33(parity) - 0.01(age) + 1.15(height) + 0.05(weight) + -8.40(smoke)
+
+```
+Part B 
+Age has a positive correlation and age has a negative correlation. 
+ 
+Part C
+Coefficient for parity might change as a result of introducing new variables like age. For example a female giving birth at a young age might have a baby at a heavier weight when compared to and older female giving birth. 
+

```

Comment:
This is a good explanation, though I'd like to see a broader explanation. 

As a random nitpick: it's sometimes a pain point in scientific writing to use "female" in contexts when you're not calling attention to sex, I'd use "mother" here. That's a random nitpick though not really relevant to the statistics.